Classes/Driver/AmazonS3Driver.php

```diff
@@ -218,7 +218,8 @@ public function processConfiguration()
      */
     public function initialize()
     {
-        $this->initializeBaseUrl()
+        $this->loadSettings()
```

Comment:
What do you think if we don't create new "loadSettings" function? We just apply code the inside the "initializeSettings", move "initializeSettings()" to the top of "initializeBaseUrl()".

```
$this->initializeSettings()
     ->initializeBaseUrl()
     ->initializeClient();
```